DetDescr/include/DetDescr/EcalGeometry.h

```diff
@@ -296,7 +194,7 @@ class EcalHexReadout : public framework::ConditionsObject {
    */
   std::vector<EcalID> getNN(EcalID id) const {
     auto list = NNMap_.at(EcalID(0, id.module(), id.cell()));
-    for (auto& flat : list) id = EcalID(id.layer(), flat.module(), flat.cell());
+    for (auto& flat : list) flat = EcalID(id.layer(), flat.module(), flat.cell());
     return list;
   }
```

Comment:
This patch is what is causing the deviation of Summed Iso between this branch and trunk.